src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1161,7 +1161,7 @@ public JSONObject getDebugState(boolean full, String endpointId)
     {
         JSONObject debugState = new JSONObject();
         debugState.put("id", id);
-        debugState.put("name", name.toString());
+        debugState.put("name", name == null ? null : name.toString());
```

Comment:
Should we use an empty string or `no name` or something instead of null?

Reply:
I don't think it makes a difference. A null value in json will make it clear that it was not set, as opposed to someone making a conference named `no name` or `null` (I'm just rationalizing my reluctance to make an update)